Sources/Fluent/Utilities/Fluent+Node.swift

```diff
@@ -1 +1,9 @@
 @_exported import Node
+
+public struct DatabaseContext: Context {
```

Comment:
Since this is passed around so much through Node, does it make more sense as a reference type?